### ISSUE TITLE
Warning System | question 

### ISSUE THREAD

**Issue by ashuezy** [AlexeyAB/darknet#703]
Hi,

I am trying to build a warning system. The objects i am looking to detect are:
-weapon
-person
-vehicle
-fire

Which model should i use yolo-tiny or yolo-v2 ? 
I have also noticed that yolov3 way too accurate. Is there an option to train on yolov3 with limited number of categories or i will have to train from scratch ? 

Also what do you mean by using pre-trained weights for the convolutional layers ? 
What part of the process is speed up by using pre-trained weights ?

**Comment by AlexeyAB** [comment]
Hi, You can use any model what you want

> Also what do you mean by using pre-trained weights for the convolutional layers ?

I mean using `darknet53.conv.74` file:
`darknet.exe detector train data/obj.data yolo-obj.cfg darknet53.conv.74`

> What part of the process is speed up by using pre-trained weights ?

Avg loss will decrease faster.

**Comment by ashuezy** [comment]
Thanks for replying.

3*2000 iterations = 6000 epochs
Can you please share your wisdom as how long will it take on my i5 7th gen 940MX 2 GB Graphics card with Cuda or i will have to buy a new machine ?

**Comment by AlexeyAB** [comment]
You can run training and measure time that is spent for 1 iteration. Then just multiply it on required itetartions.

**Comment by ashuezy** [comment]
1 min for each iteration ,,
6000 mins = 4.166667 days ! Too much.
This is for the yolov3 model.
